src/View/mainpage/MainPage.py

```diff
@@ -220,7 +220,7 @@ def create_footer(self):
         layout_footer = QtWidgets.QHBoxLayout(self.footer)
         layout_footer.setContentsMargins(0, 0, 0, 0)
 
-        label_footer = QtWidgets.QLabel("@OnkoDICOM 2019-20")
+        label_footer = QtWidgets.QLabel("@OnkoDICOM2021")
```

Comment:
shouldn't it be "@OnkoDICOM 2021"? I'm talking about the space between the word and number, that's it

Reply:
As it is in the PR, with no space, is exactly how the product owner requested it in Slack. This was also shown off during our recent meeting and no complaints were raised!

Reply:
all good then!